go.mod

```diff
@@ -13,9 +13,9 @@ require (
 	k8s.io/api v0.19.7
 	k8s.io/apimachinery v0.19.7
 	k8s.io/client-go v0.19.7
-	knative.dev/client v0.21.0
+	knative.dev/client v0.21.1-0.20210224184947-30be4b6efc6d
 	knative.dev/hack v0.0.0-20210203173706-8368e1f6eacf
```

Comment:
@rhuss @maximilien @navidshaikh 

I noticed the PR comment by `knative-automation` is to update to v0.22, is this action in our expectation?

When will this automation be triggered? Thanks!

Reply:
The goal here is to keep the related dependencies on Knative modules updated, in other words keep it releasable and verify the changes aren't breaking anything. Therefore it's using floating version to current HEAD. Similarly to client getting latest Serving, Eventing changes. Once we reach the `0.22` release cut that should be pinned to `v0.22.0`.

Reply:
The job itself is triggered twice a day, see the gh [action](https://github.com/knative-sandbox/knobots/blob/master/.github/workflows/auto-updates.yaml#L19). Also for the reference it can be enabled/disabled by adding repository to `repos.yaml` [config](https://github.com/knative-sandbox/knobots/commit/831ffede42f84e2458d2a329c5c8e403093f3bbd#diff-dbde431d2e3e7066f0e1130ea024c4e5753ec2700f333e1cbfae7822723e5d17).

Reply:
Should be 21+